assets/cart.js

```diff
@@ -6,6 +6,8 @@ class CartRemoveButton extends HTMLElement {
       event.preventDefault();
       const cartItems = this.closest('cart-items') || this.closest('cart-drawer-items');
       cartItems.updateQuantity(this.dataset.index, 0);
+
```

Comment:
This will be the wrong price when JS is off (variant change and qty amount):

https://screenshot.click/06-42-ojxav-eyfja.png

I think we hide it for now since we show it in the dropdown the price (the same way we hide the table for now and well rethink it in no JS)

cc @KaichenWang @edmund-teh